code/modules/clothing/under/ties.dm

```diff
@@ -367,3 +367,37 @@
 	icon_state = "talisman"
 	item_color = "talisman"
 	armor = list(melee = 5, bullet = 5, laser = 5, energy = 5, bomb = 20, bio = 20, rad = 5) //Faith is the best armor.
+
+//////////////
+//OBJECTION!//
+//////////////
+
+/obj/item/clothing/tie/lawyers_badge
+	name = "attorney's badge"
+	desc = "Fills you with the conviction of JUSTICE. Lawyers tend to want to show it to everyone they meet."
+	icon_state = "lawyerbadge"
+	item_color = "lawyerbadge"
+	var/mob/living/current_lawyer = null
+
+/obj/item/clothing/tie/lawyers_badge/attach(obj/item/clothing/under/U, user)
+	if(!..())
+		return 0
+	START_PROCESSING(SSobj, src)
+
+/obj/item/clothing/tie/lawyers_badge/detach(obj/item/clothing/under/U, user)
+	..()
+	STOP_PROCESSING(SSobj, src)
+	if(current_lawyer)
+		current_lawyer.bubble_icon = initial(current_lawyer.bubble_icon)
+		current_lawyer = null
+
+/obj/item/clothing/tie/lawyers_badge/process()
+	var/obj/item/clothing/under/U = loc

```

Comment:
99% of the time it will short out right below this line, is it really so costly? Accessory code isn't exactly amazing here.